client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -85,75 +102,113 @@ const customerNotifications = [
+		if ( ! nextProps.isSaving && this.props.isSaving ) {
+			if ( nextProps.submitError ) {
+				nextProps.errorNotice(
+					translate( 'There was a problem saving the email settings. Please try again.' )
+				);
+			} else {
+				nextProps.successNotice( translate( 'Email settings saved.' ), { duration: 4000 } );
+			}
+		}
 	}
 
+	onChange = event => {
+		const { onChange, siteId } = this.props;
+		onChange( siteId, event );
+	};
+
 	renderOriginNotification = ( item, index ) => {
-		const { settings, loaded, loading } = this.props;
-		return <NotificationsOrigin
-			key={ index }
-			item={ item }
-			isPlaceholder={ loading }
-			recipient={ loaded ? settings.email[ item.field ] : '' }
-			onChange={ this.recipientsChange }
-		/>;
-	}
+		const { settings, loading } = this.props;
+		return (
+			<NotificationsOrigin
+				key={ index }
+				item={ item }
+				isPlaceholder={ loading }
+				recipient={ get( settings, [ item.field, item.option, 'value' ], '' ) }
+				placeholder={ get( settings, [ item.field, item.option, 'default' ], '' ) }
+				onChange={ this.onChange }
+				checkEmail={ item.checkEmail }
+			/>
+		);
+	};
 
 	renderInternalNotification = ( item, index ) => {
-		const { settings, loaded, loading } = this.props;
-		return <InternalNotification
-			key={ index }
-			item={ item }
-			checked={ 'yes' === ( loaded && settings[ item.field ].enabled ) }
-			recipient={ loaded ? settings[ item.field ].recipient : '' }
-			isPlaceholder={ loading }
-			onToggle={ this.notificationsToggle }
-			onChange={ this.recipientsChange }
-		/>;
-	}
+		const { settings, loading } = this.props;
+		return (
+			<InternalNotification
+				key={ index }
+				item={ item }
+				checked={ 'yes' === get( settings, [ item.field, 'enabled', 'value' ], '' ) }
+				recipient={ get( settings, [ item.field, 'recipient', 'value' ], '' ) }
+				placeholder={ get( settings, [ item.field, 'recipient', 'default' ], '' ) }
+				isPlaceholder={ loading }
+				onChange={ this.onChange }
+			/>
+		);
+	};
 
 	renderCustomerNotification = ( item, index ) => {
-		const { settings, loaded, loading } = this.props;
-		return <CustomerNotification
-			key={ index }
-			item={ item }
-			isPlaceholder={ loading }
-			checked={ 'yes' === ( loaded && settings[ item.field ].enabled ) }
-			onToggle={ this.notificationsToggle }
-		/>;
-	}
+		const { settings, loading } = this.props;
+		return (
+			<CustomerNotification
+				key={ index }
+				item={ item }
+				isPlaceholder={ loading }
+				checked={ 'yes' === get( settings, [ item.field, 'enabled', 'value' ], '' ) }
+				onChange={ this.onChange }
+			/>
+		);
+	};
 
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
 		return (
 			<div className="email-settings__container">
-				<ExtendedHeader
-					label={ translate( 'Origin' ) }
-				/>
-				<List>
-					{ originNotifications.map( this.renderOriginNotification ) }
-				</List>
+				<ExtendedHeader label={ translate( 'Origin' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **74** times:
`translate( 'Original' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-settings-ui-save)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).